realm/realm-library/src/testObjectServer/kotlin/io/realm/log/obfuscator/LoginInfoObfuscatorTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.log.obfuscator
+
+import io.realm.mongodb.PatternObfuscatorFactory
+import org.junit.Test
+import kotlin.test.assertEquals
+
+const val IRRELEVANT_INPUT = """{"blahblahblah":"blehblehbleh"}"""
+
+class LoginInfoObfuscatorTest {
+
+    private val apiKeyUrlSegments = listOf("providers", "api-key")
```

Comment:
Is the ID from the Credentails not available in a JVM test?

Reply:
The credentials are located in the `objectServer` package, so they aren't visible from the main package - that's why I put the pattern factory inside objectServer so as to be able to get the ID providers, instead of using strings directly. Maybe I need to rethink where to locate these classes...